packages/side-nav/theme/lumo/vaadin-side-nav-item-styles.js

```diff
@@ -25,17 +26,9 @@ export const sideNavItemStyles = css`
 
   [part='toggle-button'] {
     position: relative;
-    border: 0;
-    margin: calc((var(--lumo-icon-size-m) - var(--lumo-size-s)) / 2) 0;
```

Comment:
This margin is a leftover from the [first iteration](https://github.com/vaadin/vcf-nav/commit/92256a9a3ace52a242accd7b6d07b926d3289ddb), and looks quite unexpected.
At least the button looks as expected without it. So let's remove this line.